compiler/src/iree/compiler/Codegen/Common/FoldTensorSubsetIntoVectorTransferOps.cpp

```diff
@@ -117,6 +119,63 @@ class FoldExtractSliceIntoTransferRead final
   }
 };
 
+/// Returns true if `writeOp` fully overwrites its destination.
+///
+/// Example:
+///
+/// ```MLIR
```

Comment:
nit: let's drop `MLIR` for consistency in this file. (side note: IMO it is a visual noise..)

```suggestion
/// ```
```